cli/cluster/instances.go

```diff
@@ -102,13 +102,22 @@ func GetInstancesConf(ctx *context.Ctx) (*InstancesConf, error) {
 	return &instancesConf, nil
 }
 
+// filterRunningMembers removes non-local members from instances map.
+func filterRunningMembers(membershipInstances *MembershipInstances, runningInstances map[string]string) {
+	for URI := range *membershipInstances {
+		if _, found := runningInstances[URI]; !found {
+			delete(*membershipInstances, URI)
+		}
+	}
+}
+
 // GetJoinedInstanceName returns a name of instancethat is already joined to cluster
```

Comment:
Hmm, now it will only return local instances. If this is ok, it should be reflected in the description and in the name of the function.